### PR TITLE
Adding examples and Formatting for the examples code

### PR DESCRIPTION
**Pull request by SeqLaz**

### CODE REVIEW

**Review by suricactus — APPROVED** [review]

File: qfieldcloud_sdk/sdk.py [hunk 1]
```diff
@@ -418,6 +441,19 @@ def upload_files(
 
         Returns:
             A list of dictionaries with information about the uploaded files.
+
+        Example:
+            ```python
+            client.upload_files(
+                project_id="123e4567-e89b-12d3-a456-426614174000",
```
Comment (suricactus) [inline qfieldcloud_sdk/sdk.py:448]:
```suggestion
                project_id="123e4567-e89b-12d3-a456-426614174000",
```
Hopefully this does not match an actual project in the DB :dagger:

Comment (SeqLaz) [reply qfieldcloud_sdk/sdk.py:448]: Hope so, too!

File: qfieldcloud_sdk/sdk.py [hunk 2]
```diff
@@ -589,6 +647,14 @@ def list_jobs(
...
         Returns:
             A list of dictionaries representing the jobs.
+
+        Example:
+        ```python
+            client.list_jobs(
+                project_id="123e4567-e89b-12d3-a456-426614174000",
+                job_type=JobTypes.PACKAGE
+            )
+        ```
```
Comment (suricactus) [inline qfieldcloud_sdk/sdk.py:657]: Wrong indentation.

File: qfieldcloud_sdk/sdk.py [hunk 3]
```diff
@@ -898,6 +1022,16 @@ def download_file(
...
+
+        Example:
+            ```python
+            client.download_file(
+                project_id="123e4567-e89b-12d3-a456-426614174000",
+                download_type=FileTransferType.PROJECT,
+                local_filename="/home/ninjamaster/QField/cloud/Tree_Survey/trees.gpkg",
+                remote_filename="trees.gpkg",
+                show_progress=True
+            )
```
Comment (suricactus) [inline qfieldcloud_sdk/sdk.py:1034]: Missing closing backticks.

Comment (SeqLaz) [reply qfieldcloud_sdk/sdk.py:1034]: Apologize!

**Pull request merged**